### PR TITLE
[Rule Tuning] Startup or Run Key Registry Modification

### PR DESCRIPTION
**Pull request by Samirbous**
## Issues

Resolves https://github.com/elastic/detection-rules/issues/1085

## Contributor checklist

- Have you signed the [contributor license agreement](https://www.elastic.co/contributor-agreement)?
- Have you followed the [contributor guidelines](https://github.com/elastic/detection-rules/blob/main/CONTRIBUTING.md)?

### CODE REVIEW

File: rules/windows/persistence_run_key_and_startup_broad.toml [hunk 1]
```diff
@@ -45,9 +44,12 @@ registry where /* length(registry.data.strings) > 0 and */
   /* add here common legit changes without making too restrictive as this is one of the most abused AESPs */
```
Comment (bm11100) [inline rules/windows/persistence_run_key_and_startup_broad.toml:44]:
```suggestion
  /* add common legitimate changes without being too restrictive as this is one of the most abused AESPs */
```

**Review by bm11100 — APPROVED** [review]

File: rules/windows/persistence_run_key_and_startup_broad.toml [hunk 2]
```diff
@@ -42,12 +41,15 @@ registry where /* length(registry.data.strings) > 0 and */
...
   not registry.data.strings : "ctfmon.exe /n" and
   not (registry.value : "Application Restart #*" and process.name : "csrss.exe") and
-  user.domain != "NT AUTHORITY" and
-  not registry.data.strings : ("C:\\Program Files\\*.exe", "C:\\Program Files (x86)\\*.exe") and
-  not process.executable : ("C:\\Windows\\System32\\msiexec.exe", "C:\\Windows\\SysWOW64\\msiexec.exe")
+  user.id not in ("S-1-5-18", "S-1-5-19", "S-1-5-20") and
+  not registry.data.strings : ("?:\\Program Files\\*.exe", "?:\\Program Files (x86)\\*.exe") and
+  not process.executable : ("?:\\Windows\\System32\\msiexec.exe", "?:\\Windows\\SysWOW64\\msiexec.exe") and
+  not (process.name : "OneDriveSetup.exe" and
+       registry.value : ("Delete Cached Standalone Update Binary", "Delete Cached Update Binary", "amd64", "Uninstall *") and
```
Comment (brokensound77) [inline rules/windows/persistence_run_key_and_startup_broad.toml:51]: :wave:, just verifying that this is supposed to be an `and` and not an `or`

**Review by brokensound77 — APPROVED** [review]
Had a question, otherwise LGTM 👍

File: rules/windows/persistence_run_key_and_startup_broad.toml (hunk 2 above)
Comment (Samirbous) [reply rules/windows/persistence_run_key_and_startup_broad.toml:51]: its an `and` to make it a bit harder to bypass the rule.

**Pull request merged**